style.css

```diff
@@ -139,18 +139,159 @@
     margin-bottom: 24.5px;
   }
 
-  .secondhorizontalline {
-    height: 0;
+  .image {
+    margin-top: 48px;
+    justify-content: center;
+    display: flex;
     width: 100%;
-    border: 1px solid #646369;
-    margin-top: 21px;
-    top: calc(50% - 0px / 2 + 0.5px);
   }
 
-  .image {
-    margin-top: 48px;
+  .workHeadingMain {
+    width: 354px;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    line-height: 52px;
+    color: #9490ff;
+    margin: 80px 24px 33px 24px;
+  }
+
+  .workGridContainer {
+    width: 354px;
+    margin: 40px 24px 0 24px;
+    display: grid;
+    grid-template-columns: auto;
     justify-content: center;
+  }
```

Comment:
- [ ] There is a strange margin to the right of your content that is not present in the template design, please kindly fix it 🙃 🙃

Your design
<img width="327" alt="Screen Shot 2022-08-19 at 13 57 46" src="https://user-images.githubusercontent.com/74501523/185619938-8ff4efbe-f76a-4e2c-b444-ef9646c0b74b.png">

Figma design
<img width="374" alt="Screen Shot 2022-08-19 at 13 56 27" src="https://user-images.githubusercontent.com/74501523/185620103-c0162a1f-8c4e-4cd8-b6d9-9b8a06bc5905.png">